velox/dwio/parquet/writer/Writer.h

```diff
@@ -109,6 +109,7 @@ struct WriterOptions : public dwio::common::WriterOptions {
   /// Timestamp unit for Parquet write through Arrow bridge.
   /// Default if not specified: TimestampUnit::kNano (9).
   std::optional<TimestampUnit> parquetWriteTimestampUnit;
+  std::optional<std::string> parquetWriteTimestampTimeZone;
```

Comment:
can we add a quick documentation explaining how this is used and listing the default value?

Reply:
@pedroerp Yes. Added.